src/main/webapp/WEB-INF/tags/student/feedbackResults/questionWithResponses.tag

```diff
@@ -12,8 +12,14 @@
 
     <c:if test="${questionWithResponses.questionDetails.individualResponsesShownToStudents}">
 
-      <c:forEach items="${questionWithResponses.responseTables}" var="responseTable">
-        <feedbackResults:responseTable responseTable="${responseTable}"/>
+      <c:forEach items="${questionWithResponses.othersResponseTables}" var="othersResponseTable">
+        <feedbackResults:responseTable responseTable="${othersResponseTable}"/>
+      </c:forEach>
+
+      <h4>Your own responses:</h4>
```

Comment:
I think it might be better to make this heading a little smaller than its current size. This is because right now it is the same size as the question, so it looks like this:

<img width="1156" alt="screen shot 2018-03-30 at 5 59 41 pm" src="https://user-images.githubusercontent.com/24493463/38134167-76d856d4-3444-11e8-981d-5d3f4d4a9265.png">

But I think it can be slightly smaller to distinguish it further from the question.